### ISSUE TITLE
tlogreplay : fix WriteZeroesAt offset/length

### ISSUE THREAD

**Issue by iwanbk** [g8os/blockstor#118]
see the discussion here https://github.com/g8os/blockstor/pull/115#discussion_r115004190
- it has wrong length
- it was not tested, executing `dd if=/dev/zero ....` doesn't make `WriteZeroesAt` to be called.

Instead of storing full block to tlog, it seems we can simply store the []byte received by WriteAt and WriteZeroesAt

**Comment by iwanbk** [comment]
```
it was not tested, executing dd if=/dev/zero .... doesn't make WriteZeroesAt to be called.
```
From my code reading, WriteZeroesAt is not supported by nbd-client

**Comment by iwanbk** [comment]
Fixed at f096bf67317649afaf0926408a8b50b709d6cf34
